project/src/main/webapp/index.html

```diff
@@ -24,8 +25,60 @@
     	</div>
       </nav>
     <div class="container">
-      <h1>Covidiena</h1>
-      <p>This is our project.</p>
+      <div class="section">
+        <h4>Our Vision</h4>
+        <p>With the recent pandemic interrupting the education of millions of students throughout the country, colleges and universities are struggling to find ways to let students back on campus. That’s where Covidiena comes in. Our platform aims at easing the transition to in-person learning for college students, staff, and faculty during the ongoing crisis so students can continue their education in a stable and safe environment.</p>
+      </div>
+      <div class="card-panel blue-grey lighten-4">
+        <h4>What We Do</h4>
+        <div class="row">
+          <div class="col s4">
```

Comment:
maybe s12 m4? up to you style-wise at the end of the day, same for the other two cards.

Reply:
Are these classes for mobiles and slim screens? I'll put that in for sure!

Reply:
So if you only define s12 or just s4, it will be full width or thirds on small screens and up automatically, respectively. But you can put s12 m4, and it will be full width on small screens, thirds on medium screens and up.